libdispatch/dutil.c

```diff
@@ -281,7 +281,7 @@ NC_readfileF(FILE* stream, NCbytes* content, long long amount)
 {
     int ret = NC_NOERR;
     long long red = 0;
-    char part[1024];
+    char part[4194304];
```

Comment:
Can we get a pre-processor define for this number?

Reply:
Usually defining a C macro is so it can be reused multiple times.
But feel free to make a change.